### PR TITLE
Add DCH to OVA

### PR DESCRIPTION
**Pull request by andrewtchin**
Fixes https://github.com/vmware/vic/issues/5463
screen shot 2017-08-30 at 6 15 08 pm

### CODE REVIEW

**Comment by mdubya66** [comment]
Forgive my ignorance but does this happen during OVA build time or during OVA install?

**Comment by andrewtchin** [comment]
during build time

File: installer/packer/scripts/system_settings.sh [hunk 1]
```diff
@@ -41,6 +41,7 @@ systemctl enable engine_installer_startup.service engine_installer.service
 
 # Clean up temporary directories
 rm -rf /var/tmp/harbor
```
Comment (anchal-agrawal) [inline installer/packer/scripts/system_settings.sh]: This line can be nuked.

Comment (andrewtchin) [reply installer/packer/scripts/system_settings.sh]: lol thanks fixed

**Pull request merged**